packages/elements/src/components/ino-chip/ino-chip.scss

```diff
@@ -2,63 +2,63 @@
 @use '@material/chips';
 @use "@material/chips/styles";
 @use 'base/theme';
+//@use 'base/new-theme' as theme;
 
-$default-chip-color: lighten(theme.color(primary), 36.5%);
-$default-chip-contrast-color: rgba(0, 0, 0, 0.87);
-$default-chip-dark-color: rgba(0, 0, 0, 0.87);
+$n-3: #7275A1;
+$n-7: #EDEFF5;
+$primary: #2C02FF;
+$border-focused: #2d5dff4d;
```

Comment:
Is this a rgba color? Because it has more than 6 chars. If yes than I prefer the rgba-Syntax. 

Generally we should discuss if we want to have an consistent color-code-stlye (hsl, rgb, hex, ...)